plugins/french/xiaowaz.ts

```diff
@@ -85,35 +107,35 @@ class XiaowazPlugin implements Plugin.PluginBase {
     const PARAGRAPH_EXCLUDE_LIST = [
       'Écrit par',
       'Ecrit par',
-      'Sorties régulières le mardi et vendredi\u00A0!',
+      'Sorties régulières',
       'Auteur\u00A0:',
       'Statut VO\u00A0:',
       'Nom utilisé\u00A0:',
       'Auteur original\u00A0:',
       'Auteur original de l’oeuvre',
       '851 chapitres en',
+      'Index\u00A0:',
     ];
 
-    $('.entry-content p').each((index, element) => {
+    $('.entry-content > p').each((index, element) => {
+      // remove chapter links
+      if ($(element).find('a[href*="xiaowaz.fr/articles"]').length !== 0)
+        return;
+
       const balise = $(element);
       const textbalise = balise.text();
-      if (
-        PARAGRAPH_EXCLUDE_LIST.some(keyword => textbalise.includes(keyword))
-      ) {
-        return false;
-      }
+
+      if (PARAGRAPH_EXCLUDE_LIST.some(keyword => textbalise.includes(keyword)))
+        return;
+
       if (
         !textbalise.includes('Genre') &&
         !textbalise.includes('Synopsis') &&
-        //Managing the novel 'Rebirth of The Thief Who Roamed The World' to remove these three fields.
-        !textbalise.includes('重生之賊行天下') &&
```

Comment:
it was to avoid seeing the title of the novel in the summary

Reply:
i forgot i removed it... oops